src/transaction.ts

```diff
@@ -283,6 +282,29 @@ export async function performAtomicTransactionComposerDryrun(atc: AtomicTransact
   return new algosdk.DryrunResult(await algod.dryrun(dryrun).do())
 }
 
+/**
+ * Performs a simulation of the transactions loaded into the given AtomicTransactionComposer.
+ * @param atc The AtomicTransactionComposer with transaction(s) loaded.
+ * @param algod An Algod client to perform the simulation.
+ * @returns The simulation result, which includes various details about how the transactions would be processed.
+ */
+export async function performAtomicTransactionComposerSimulate(atc: AtomicTransactionComposer, algod: Algodv2) {
+  const signedTransactions = await atc.gatherSignatures()
```

Comment:
Do we need them to be signed transactions for simulate (I'm not sure)?

Reply:
We don't. I've sent @negar-abbasi a snipped of how signatures can be ignored she will push it later on, we will also smoke test it with real wallet just to be sure